src/app/_components/SocialLinks/index.tsx

```diff
@@ -1,15 +1,35 @@
 import Link from 'next/link'
 
-import { LinkedInIcon } from '../../_icons/socialIcons'
+import { LinkedInIcon, TwitterIcon } from "../../_icons/socialIcons";
 
 export default function SocialLinks({ socials }) {
   return (
     <div style={{ marginTop: '10px' }}>
-      {socials.map((link, i) => (
-        <Link href={link} key={i}>
+      {socials.linkedIn.length === 1 && (
+        <Link href={socials.linkedIn[0].url} target={'_blank'} rel="noopener noreferrer">
           <LinkedInIcon />
         </Link>
-      ))}
+      )}
+
+      {socials.x.length === 1 && (
```

Comment:
Why are these arrays? Can users have multiple versions of each Social profile?

Reply:
users cannot have multiple versions of each social profile, but socials are now part of a "tab" and consequently stored as an array called "Socials":

![imagem](https://github.com/user-attachments/assets/89fc7012-46fd-4361-845d-809a10f289f4)